conjureup/ui/views/bootstrapwait.py

```diff
@@ -38,7 +38,7 @@ def redraw_kitt(self):
                                             'bootstrap.err')
         out = check_output("tail -n 10 {}".format(bootstrap_stderrpath),
                            shell=True)
-        self.output.set_text(out)
+        self.output.set_text(out.stderr.decode())
```

Comment:
This was causing the snappy run to error out with a traceback, something about the encoding of the output was screwing it up.